dlhub_sdk/client.py

```diff
@@ -128,39 +128,40 @@ def get_task_status(self, task_id):
         r = self.get("{task_id}/status".format(task_id=task_id))
         return r.json()
 
-    def describe_servable(self, owner, name):
+    def describe_servable(self, name):
         """Get the description for a certain servable
 
         Args:
-            owner (string): Username of the owner of the servable
-            name (string): Name of the servable
+            name (string): DLHub name of the servable of the form <user>/<servable_name>
         Returns:
             dict: Summary of the servable
         """
+        split_name = name.split('/')
+        if len(split_name) < 2:
+            raise AttributeError('Please enter name in the form <user>/<servable_name>')
 
         # Create a query for a single servable
```

Comment:
I'm not sure what the `-1*len(split_name)-1)...` does. If you want the portions of the name that are the "dlhub.name" not the "dlhub.owner," could you just do `splitname[1:]`?